### PR TITLE
Some animations

### PR DESCRIPTION
**Pull request by SBird1337**
Add some new animations. Also adds relevant graphics and tags them.

* Pluck
* Feint
* Trump Card
* Acupressure
* Wring Out
* Power Swap
* Guard Swap
* Worry Seed

### CODE REVIEW

File: data/battle_anim_scripts.s [hunk 1]
```diff
@@ -134,7 +134,7 @@ gBattleAnims_Moves::
 	.4byte Move_FOCUS_ENERGY
 	.4byte Move_BIDE
 	.4byte Move_METRONOME
-	.4byte Move_MIRROR_MOVE @ doesn�t have an actual animation
+	.4byte Move_MIRROR_MOVE @ doesn�t have an actual animation
```
Comment (DizzyEggg) [inline data/battle_anim_scripts.s]: What's that?

File: src/graphics.c [hunk 2]
```diff
@@ -114,6 +116,9 @@ const u32 gBattleAnimSpritePal_Hit2[] = INCBIN_U32("graphics/battle_anims/sprite
 const u32 gBattleAnimSpritePal_WavingHand[] = INCBIN_U32("graphics/battle_anims/sprites/waving_hand.gbapal.lz");
 const u32 gBattleAnimSpriteGfx_WavingHand[] = INCBIN_U32("graphics/battle_anims/sprites/waving_hand.4bpp.lz");
 
+const u32 gBattleAnimSpriteGfx_Feint[] = INCBIN_U32("graphics/battle_anims/sprites/feint-punch.4bpp.lz");
+const u32 gBattleAnimSpritePal_Feint[] = INCBIN_U32("graphics/battle_anims/sprites/feint-punch.gbapal.lz");
```
Comment (DizzyEggg) [inline src/graphics.c]: Could you use underscores instead of dashes?

File: data/battle_anim_scripts.s (hunk 1 above)
Comment (SBird1337) [reply data/battle_anim_scripts.s]: thats text encoding for you... resolved in f1c82abc7e01c2e617d00ad3e7f789a8b02de477

File: src/graphics.c (hunk 2 above)
Comment (SBird1337) [reply src/graphics.c]: sure: f1c82abc7e01c2e617d00ad3e7f789a8b02de477

File: src/graphics.c [hunk 3]
```diff
@@ -119,11 +119,11 @@ const u32 gBattleAnimSpriteGfx_WavingHand[] = INCBIN_U32("graphics/battle_anims/
 const u32 gBattleAnimSpriteGfx_Feint[] = INCBIN_U32("graphics/battle_anims/sprites/feint-punch.4bpp.lz");
```
Comment (SBird1337) [inline src/graphics.c]: this needs to be done still

File: src/battle_anim_effects_1.c [hunk 4]
```diff
@@ -2636,6 +2841,252 @@ static void AnimHyperBeamOrbStep(struct Sprite* sprite)
...
+    sprite->pos1.x += gBattleAnimArgs[0];
+    sprite->pos1.y += gBattleAnimArgs[1];
+    sprite->callback = AnimPluckParticle;
+}
+
+static void AnimMoveFeintSwipeStep(struct Sprite* sprite)
+{
+    switch(sprite->data[5])
+    {
+        case 0:
```
Comment (DizzyEggg) [inline src/battle_anim_effects_1.c]: Small nitpick, but could you have the cases on the same line as switch?

Comment (SBird1337) [reply src/battle_anim_effects_1.c]: bcffc2655e0401bb129787d39f60137be1a9575c

File: src/graphics.c (hunk 3 above)
Comment (SBird1337) [reply src/graphics.c]: bcffc2655e0401bb129787d39f60137be1a9575c

**Comment by SBird1337** [comment]
I will probably add some more animations, so idk if you want to merge those now and I'll do another PR or you want to hold off on that, in any case: There are a lot of commits that only fix style stuff or hiccups I produced myself, do you want me to squash the fix commits, or all commits, or nothing at all?

**Pull request merged**